web/home.go

```diff
@@ -13,8 +13,9 @@ type HomeData struct {
 
 func HomeHandler(c *gin.Context) {
 	data := HomeData{
-		Title:     defaultLayoutData.Title,
-		Paragraph: "This is the home page",
+		Title: defaultLayoutData.Title,
```

Comment:
As we already put all the data in the template, I would vote to pass the `title` and `paragraph` texts there too, without having them here in the controller code